query_optimizer/tests/execution_generator/Select.test

```diff
@@ -953,19 +953,79 @@ WHERE double_col < 0
 ==
 
 # Window Aggregation Test.
-# Currently this is not supported, an empty table will be returned.
-SELECT avg(int_col) OVER w FROM test
+SELECT char_col, long_col, avg(long_col) OVER w FROM test
 WINDOW w AS
-(PARTITION BY char_col
- ORDER BY long_col DESC NULLS LAST
+(ORDER BY char_col DESC NULLS LAST
```

Comment:
I get it, but do we have other alternatives to keep the semantics of `window aggregation`: aggregating over incoming tuples.

It particular makes more sense for the following next test cases w/ `NULL`s.